src/Engine.cpp

```diff
@@ -74,11 +82,13 @@ void Engine::handleEvents() {
 }
 
 void Engine::update() {
-    manager.refresh();
-    manager.update();
+    Manager *man = levelloader->getManager();
```

Comment:
Don't create a pointer reference and assign it in the update loop

Reply:
Fixed, gone in latest push to branch